spec/support/reporters/nyc-test-coverage-reporter.mjs

```diff
@@ -0,0 +1,32 @@
+// This file runs in node.js.
+
+import fsPromises from 'node:fs/promises';
+import {URL} from 'node:url';
+import path from 'node:path';
+
+export default class NycTestCoverageReporter {
+  // The following method runs after every Jasmine describe block.
+  suiteDone({properties}) {
+    this.coverage = properties?.__coverage__ || this.coverage;
+  }
+
+  jasmineDone() {
+    const rootDirname = path.dirname(
+      new URL(import.meta.resolve('#root/package.json')).pathname
```

Comment:
Looking at the documentation in Node it appears that `import.meta.resolve` is marked as experimental, [Stability: 1.2](https://nodejs.org/docs/latest/api/documentation.html#stability-index) - Release candidate. It appears the minimum node version required to use this is 18.20.4, using a version less than this produces the error `TypeError: (intermediate value).resolve is not a function`. Could there be any issues in using this?

Reply:
We control the local node version. `import.meta.resolve` is a part of html standard, not node, so there is no risk of this feature being removed or changed: https://html.spec.whatwg.org/multipage/webappapis.html#integration-with-the-javascript-module-system

Reply:
I was thinking are not running it in a node context here? and if so does the statement from MDN apply:

> Node.js also implements [the import.meta.resolve() function](https://nodejs.org/docs/latest/api/esm.html#importmetaresolvespecifier), but adds an additional parent parameter if you use the --experimental-import-meta-resolve flag.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import.meta/resolve#not_an_ecmascript_feature

Reply:
This relates to an additional parameter to resolve() named "parent". We don't use this experimental feature in this code.

Reply:
Thanks for clarifying 👍